specification/Metadata/0.0.0/schema/textureAccessor.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "A description of how to access data values from the color channels of a texture",
```

Comment:
`"title"` should just be "Texture Accessor" and this summary should go in a `"description"` property. The description should also include the note that the method of selecting a texture is implementation-defined.